app/views/taxons/sections/_news_and_communications.html.erb

```diff
@@ -18,18 +22,25 @@
         <% end %>
       </div>
     </div>
+    <% if no_more_news_items %>
+      <%= link_to(
+        section[:see_more_link][:text],
+        section[:see_more_link][:url], class: "taxon-page__featured-see-more"
+      ) %>
+    <% end %>
   </div>
 
+  <% if !no_more_news_items %>
```

Comment:
I would remove the double-negative, and instead change this to an `else` statement.

Reply:
Good point, I've kept the if/if structure the same but renamed the variable to `documents`. So it now checks for `documents` or `!documents`, which I think is clearer?